onnxruntime/core/graph/contrib_ops/quantization_defs.cc

```diff
@@ -184,25 +184,34 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
 static const char* DequantizeLinear_ver1_doc = R"DOC(
 The linear dequantization operator. It consumes a quantized data, a scale, a zero point and computes the full precision data.
 The dequantization formula is y = (x - x_zero_point) * x_scale.
-Scale and zero point must have same shape. They must be either scalar (per tensor) or 1-D tensor (per 'axis').)DOC";
+x_scale and x_zero_point must have same shape. They must be scalar (per tensor), 1-D tensor (per 'axis'), or have a
+rank identical to the input (blocked quantization).)DOC";
```

Comment:
should I update the [markdown file](https://github.com/microsoft/onnxruntime/blob/main/docs/ContribOperators.md#commicrosoftdequantizelinear)?

Reply:
yes, or it won't pass the CI:
https://github.com/microsoft/onnxruntime/blob/main/tools/python/gen_contrib_doc.py
https://github.com/microsoft/onnxruntime/blob/main/tools/python/gen_opkernel_doc.py

Reply:
I also noticed that using --gen_doc to trigger build.py will build the docs using the smae py files.

Reply:
updated md docs

Reply:
you don't need to update the q/dq definition here. It is contrib op for internal use. we only need to support the standard version.